node/src/Commands.ts

```diff
@@ -2305,6 +2305,41 @@ export function createXRead(
     return createCommand(RequestType.XRead, args);
 }
 
+/**
+ * Represents a the return type for XInfo Stream in the response
+ */
+export type ReturnTypeXinfoStream = {
```

Comment:
Should these exported types also be included in index.ts?

Reply:
yes ❤️